udata/tasks.py

```diff
@@ -53,6 +53,18 @@ def get_logger(name):
     return logger
 
 
+def connect(signal):
+    def wrapper(func):
+        t = task(func)
+
+        def call_task(issue, **kwargs):
```

Comment:
Not issue specific anymore. The parameter should be something more generic like ``subject``, ``item``...